ci/docker/os-image-stemcell-builder-bionic/Dockerfile

```diff
@@ -24,25 +24,47 @@ RUN apt-get update \
     lsof \
   && apt-get clean
 
+# Nokogiri dependencies
+RUN apt-get install -y libxslt-dev libxml2-dev && apt-get clean
+
 RUN groupadd -o -g ${GROUP_ID} ubuntu                  \
   && useradd -u ${USER_ID} -g ${GROUP_ID} -m ubuntu \
   && echo 'ubuntu ALL=NOPASSWD:ALL' >> /etc/sudoers
 
-# Nokogiri dependencies
-RUN apt-get install -y libxslt-dev libxml2-dev && apt-get clean
+# install ruby
+RUN \
+  wget -O ruby-install-0.8.3.tar.gz https://github.com/postmodern/ruby-install/archive/v0.8.3.tar.gz \
+  && tar -xzvf ruby-install-0.8.3.tar.gz \
+  && cd ruby-install-0.8.3/ \
+  && make install \
+  && cd .. \
+  && rm -r ruby-install* \
+  && ruby-install 3.0.2
```

Comment:
We install ruby now without providing a sha and the `--cleanup` flag as it is done in the install-ruby.sh [here](https://github.com/cloudfoundry/bosh-linux-stemcell-builder/blob/bb1b11507c495d453fe0fb4bf617b8b2aa31b3a6/ci/docker/os-image-stemcell-builder/install-ruby.sh#L34). Additionally, we don't install bundler any more like it was done [before](https://github.com/cloudfoundry/bosh-linux-stemcell-builder/blob/bb1b11507c495d453fe0fb4bf617b8b2aa31b3a6/ci/docker/os-image-stemcell-builder/install-ruby.sh#L40-L41). Is there a reason for these changes?

Reply:
i took this way of installing ruby from the jammy stemcells.
- bundler is now installed by default with using these ruby installer

Reply:
Then I think we should add sha checking for both instead of dropping it in both places.